src/test/java/testapp/core/AccountValidatorTest.java

```diff
@@ -0,0 +1,90 @@
+package testapp.core;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalMemcacheServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import testapp.modules.AppModule;
+
+import java.util.List;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
+public class AccountValidatorTest {
+    private Injector injector = Guice.createInjector(new AppModule());
+    private AccountRepository accountRepository = injector.getInstance(AccountRepository.class);
+
+    private LocalServiceTestHelper helper = new LocalServiceTestHelper(
+            new LocalMemcacheServiceTestConfig(), new LocalDatastoreServiceTestConfig());
+
+
+    @Before
+    public void setUp() throws Exception {
+        helper.setUp();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        helper.tearDown();
+    }
+
+    @Test
+    public void registerAccount() throws Exception {
+        accountRepository.registerAccount(new UserAccount(1));
+
+        UserAccount account1 = accountRepository.findUserAccount(1);
```

Comment:
this seems like another integration test, is there unit test for this functionality